src/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
+    <script
+      src="https://kit.fontawesome.com/de9dd03ce2.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+    <section class="menu-section">
+        <div class="first-nav">
+            <h2>Today list to do</h2>
+            <i class="arrows fa-solid fa-rotate"></i>
+        </div>
+        <hr>
+        <div class="the-input-bar">
+            <input class="input-bar" type="text" placeholder = "Add to your list" required/>
+            <i class="fa-solid fa-circle-plus"></i>
+        </div> 
+        
+        <div class="list-elements">
+            <!-- <div class="the-elements">
+                <input type="checkbox">
+                <p>go to gym</p>
+            </div>
+            <hr> -->
```

Comment:
- [ ] Kindly remove these commented codes to make your codebase clean and lean. It's a good practice to not have commented code in your codebase. Commenting is about writing steps, pseudocode in your code, or small documentation in your code for later reference for you or another developer. Therefore, unused codes commented out is really not a comment but makes your codebase dirty. Please remove it 👍
- [ ] Please design your To-Do app just like the interface given in the project brief. The only thing missing is that you should give a white background color to the app. You can see I drew a red box on the wrapper you should have the white background color. 👍 

![image](https://user-images.githubusercontent.com/50941074/177413062-9455d56d-9a7a-466b-89d5-1f42fb6cbe6a.png)

Reply:
done